health-sdk/sdk/src/main/java/net/gini/android/health/sdk/review/ReviewFragment.kt

```diff
@@ -609,13 +623,28 @@ class ReviewFragment private constructor(
         }
     }
 
+    private fun createSharePendingIntent() =  PendingIntent.getBroadcast(
+        requireContext(), CHOOSER_REQUEST_ID,
+        Intent(requireContext(), ShareWithBroadcastReceiver::class.java),
+        PendingIntent.FLAG_MUTABLE or PendingIntent.FLAG_UPDATE_CURRENT
```

Comment:
[Suggestion] According to the [documentation](https://developer.android.com/reference/android/app/PendingIntent#FLAG_MUTABLE) _"It is strongly recommended to use [FLAG_IMMUTABLE](https://developer.android.com/reference/android/app/PendingIntent#FLAG_IMMUTABLE) when creating a PendingIntent."_. Is there something forcing us to use `FLAG_MUTABLE`?

Reply:
Nope, there's no need for it to be mutable. Updated in the latest commit.